packages/ui/src/helpers/accountSettings/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { AmplifyUser } from '../../types';
+import { Auth } from 'aws-amplify';
+
+type ChangePasswordInput = {
+  user: AmplifyUser;
+  currentPassword: string;
+  newPassword: string;
+};
+
+export const changePassword = async ({
+  user,
+  currentPassword,
+  newPassword,
+}: ChangePasswordInput) => {
```

Comment:
TS noob question, should we type the return of this function?

Reply:
I personally like to have them be auto-inferred for simple functions like this, but it's debatable. 🤔 

Either way, it gets TS completion.

Reply:
Actually, I think it's beneficial to have return annotation because so that other readers don't have to know what `Auth.changePassword` does or returns. I'll add a return type!

Reply:
Done in [05472b6](https://github.com/aws-amplify/amplify-ui/pull/2885/commits/05472b6582c90f19b53f6165062450415fb1a779)! Did a little bit more and added a wrapper around what `Auth.changePassword` returns.